check_license/file-exceptions.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh -e
+# Copyright (C) 2024 Intel Corporation
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# You can add an exception file
+grep -v -E -e 'src/uthash/.*' \
+           -e 'benchmark/ubench.h' \
+           -e 'include/umf/proxy_lib_new_delete.h' \
+           -e 'scripts/docs_config/conf.py' \
```

Comment:
docs files moved to a new `<umf_root>/docs` dir, so at least this line should be updated